exonum-java-binding/core/src/main/java/com/exonum/binding/core/blockchain/BlockchainProofs.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.core.blockchain;
+
+import com.exonum.binding.core.storage.database.Snapshot;
+import com.exonum.binding.core.storage.database.View;
+import com.exonum.binding.core.util.LibraryLoader;
+import com.exonum.core.messages.Proofs.BlockProof;
+import com.exonum.core.messages.Proofs.IndexProof;
+import com.google.protobuf.InvalidProtocolBufferException;
+
+/**
+ * Provides constructors of block and index proofs.
+ */
+final class BlockchainProofs {
+
+  static {
+    LibraryLoader.load();
+  }
+
+  /**
+   * Creates a block proof for the block at the given height.
+   * @param view a database view
+   * @param height the height of the block
+   */
+  static BlockProof createBlockProof(
+      /* todo: here snapshot is not strictly required — but shall we allow Forks (see the ticket) */
```

Comment:
Actually I couldn't find any use case for block proofs in execution context (where fork is available), so I propose to keep such restriction for now.